yarn-project/end-to-end/src/guides/up_quick_start.sh

```diff
@@ -4,48 +4,71 @@
 set -eux
 
 # docs:start:declare-accounts
-ALICE="0x2e13f0201905944184fc2c09d29fcf0cac07647be171656a275f63d99b819360"
-BOB="0x0d557417a3ce7d7b356a8f15d79a868fd8da2af9c5f4981feb9bcf0b614bd17e"
+ALICE="0x296904eaff80711ddd5fd39d75920bfc22725cc0ae3089f16a09a4ceb91be141"
+BOB="0x1106245c5c50798338e55094c38025db4053a25e96dd8cbc1e87c6dcbfb9d2ba"
+ALICE_PRIVATE_KEY="0x2153536ff6628eee01cf4024889ff977a18d9fa61d0e414422f7681cf085c281"
 # docs:end:declare-accounts
 
 # docs:start:deploy
 aztec-cli deploy \
-  PrivateTokenContractAbi \
-  --args 1000000 $ALICE \
+  TokenContractAbi \
   --salt 0
+
+aztec-cli check-deploy --contract-address 0x2d23acefa3ce07b3c308caf78d86c064cdf8957bcea48b38753cf58441796c8c
+
+aztec-cli send _initialize \
+  --args $ALICE \
+  --contract-abi TokenContractAbi \
+  --contract-address 0x2d23acefa3ce07b3c308caf78d86c064cdf8957bcea48b38753cf58441796c8c \
```

Comment:
probably worth making the address a param if its used alot / fixed. especially since its in the docs

Reply:
Josh has a [PR](https://github.com/AztecProtocol/aztec-packages/pull/2344#pullrequestreview-1630017582) open for this and we will probably nuke this file so I think it's ok to not do that change here.

Reply:
As mentioned in https://github.com/AztecProtocol/aztec-packages/pull/2304#discussion_r1328440822, nuke it.